src/Uno.UI.Wasm/ts/Windows/Storage/FileSavePicker.ts

```diff
@@ -0,0 +1,23 @@
+﻿namespace Windows.Storage.Pickers {
+
+	export class FileSavePicker {
+		public static SaveAs(fileName: string, dataPtr: any, size: number): void {
+
+			var buffer = new Uint8Array(size);
```

Comment:
Codacy found an issue: [Identifier 'buffer' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5702699)